.github/workflows/integration_test.yaml

```diff
@@ -16,3 +16,4 @@ jobs:
       modules: '["test_charm", "test_nginx", "test_s3", "test_redis"]'
       self-hosted-runner: true
       self-hosted-runner-label: "edge"
+      tmate-debug: true
```

Comment:
Shouldn't this be set back to false now ?

Reply:
I can remove it in a new PR, I honestly feel this will be needed for a while though :)